src/CodeGen_Internal.cpp

```diff
@@ -391,16 +391,18 @@ void get_target_options(const llvm::Module &module, llvm::TargetOptions &options
     get_md_string(module.getModuleFlag("halide_mcpu"), mcpu);
     get_md_string(module.getModuleFlag("halide_mattrs"), mattrs);
 
+    bool allow_strict_float = false;
```

Comment:
maybe a nit, "allow" seems like a confusing wording here: at this point we either use strict or don't, there's no decision left to be made as "allow" would suggest.